tests/tier1/tc_1052_check_hypervisor_id_option_in_etc_virtwho_d.py

```diff
@@ -62,7 +62,7 @@ def test_run(self):
                 host_display = self.stage_consumer_get(self.ssh_host(), register_config,
                                         host_name, host_uuid)
                 host_display_name = host_display['name']
-            if hypervisor_display in host_display_name:
+            if hypervisor_display or hypervisor_display.lower() in host_display_name:
```

Comment:
Is it caused by new satellite version?
So if the new satellite display uuid.lower() for all hypervisors now?

Reply:
Seems the satellite always display with the lower case letters.
For our ci env, the hwuuid for rhevm and hyperv modes are using upper case letters, so we need the `str.lower()` for them.

Reply:
My confusion is why the case was not failed before, so I'm thinking about has something changed.
And the hwuuid should be only supported for esx and rhevm.
```
# man virt-who-config
       hypervisor_id
              Property that should be used as identification of the hypervisor. Can be one of following: uuid, hostname, hwuuid. Note that some virtualization back‐
              ends don't have all of them implemented. Default is uuid. hwuuid is applicable to esx and rhevm only. This property is meant to be set up before  ini‐
              tial run of virt-who. Changing it later will result in duplicated entries in the subscription manager.
```

Reply:
ah...OK...
My confusion is why the case was not failed before -> we add the satellite display check recently :)

Reply:
And I was wrong, the hypev mode failed when test it with `hypervisorid=hostname`:
the hypervisorid in the mapping info is: `"hypervisorId": "WIN-UK46OOG1H9E"`, but shown the `virt-who-win-uk46oog1h9e-1` on the Satellite UI

Reply:
Got it, thanks for adding the new checking.